packages/strapi-design-system/src/Field/FieldInput.js

```diff
@@ -20,7 +20,7 @@ const Input = styled.input`
   font-size: ${14 / 16}rem;
   display: block;
   width: 100%;
-
+  height: 100%;
```

Comment:
What was the issue with the height ?

Reply:
If you look at my description and the pictures I attached, the inner input tag's height doesn't fill the outer div container, therefore when you have a field with previous autocomplete fields it gives the input a blue color, but it looks a bit off to me as it is now: 

![image](https://user-images.githubusercontent.com/93989197/147272205-4502f9cf-fcbf-4da2-91d8-962107351154.png)

This is the result with height 100%:
![image](https://user-images.githubusercontent.com/93989197/147272305-2f97ca82-4519-43cc-9bb4-512e7788e0a8.png)

Reply:
Great! Thanks for that!